indico/http/serialization.py

```diff
@@ -48,6 +44,10 @@ def deserialize(response, force_json=False):
         )
 
 
+
+def raw_bytes(content, *args, **kwargs):
+    return content
```

Comment:
Where is this used?

Reply:
https://github.com/IndicoDataSolutions/indico-client-python/pull/213/files#diff-ef5fa222e2bc2fb27c766b726185367b39ec9ef84ca3ac887ebec44f0b2dc7fcR74